wsgi/odyssy/people/models.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import unicode_literals
+from django.core.urlresolvers import reverse
+
+from django.db import models
+
+
+class People(models.Model):
+    class Meta:
+        ordering = ('name',)
+        verbose_name = 'People'
+        verbose_name_plural = 'People'
+
+    STATUS_CHOICES = (
+        ('faculty', 'Faculty'),
+        ('staff', 'Staff'),
+        ('visiting_faculty', 'Visiting Faculty'),
+    )
```

Comment:
Shouldn't `Director, Registrar` and other administrative staff be included in the `STATUS_CHOICES` because in the other app, we can use them as foreign key for other informations.